### PR TITLE
feat(suite): #4039 fix acquire desktop subtitles

### LINKED ISSUE #4039: Improve troubleshooting step in onboarding
In the desktop app, it would be more fitting to suggest that a user quits and reopens their app rather than "refreshes". 

If a user is on desktop, please change the troubleshooting subtitle to: "After closing other browser tabs and windows, try quitting and reopening the Suite app."

image (3)

### PR DESCRIPTION
**Pull request by marekrjpolak**
closes #4039
image

### CODE REVIEW

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { Button } from '@trezor/components';
 import { Translation, TroubleshootingTips } from '@suite-components';
 import { useDevice, useActions } from '@suite-hooks';
 import * as suiteActions from '@suite-actions/suiteActions';
+import { isDesktop } from '@suite/utils/suite/env';
```
Comment (tomasklim) [inline packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]:
```suggestion
import { isDesktop } from '@suite-utils/env';
```

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx [hunk 2]
```diff
@@ -36,12 +38,28 @@ const DeviceAcquire = () => {
...
+                        ),
                     },
                     {
                         key: 'device-reconnect',
                         heading: <Translation id="TR_RECONNECT_YOUR_DEVICE" />,
-                        description: <Translation id="TR_RECONNECT_DEVICE_DESCRIPTION" />,
+                        description: (
+                            <Translation
+                                id={
+                                    desktop
```
Comment (tomasklim) [inline packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]:
```suggestion
                                    isDesktop()
```

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx [hunk 3]
```diff
@@ -17,6 +18,7 @@ const DeviceAcquire = () => {
     const { acquireDevice } = useActions({
         acquireDevice: suiteActions.acquireDevice,
     });
+    const desktop = isDesktop();
```
Comment (tomasklim) [inline packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]:
```suggestion
```

File: packages/suite/src/support/messages.ts [hunk 4]
```diff
@@ -6352,6 +6357,11 @@ const definedMessages = defineMessages({
         defaultMessage: 'After closing other tabs and windows, try refreshing this page.',
         id: 'TR_TROUBLESHOOTING_CLOSE_TABS_DESCRIPTION',
     },
+    TR_TROUBLESHOOTING_CLOSE_TABS_DESCRIPTION_DESKTOP: {
+        defaultMessage:
+            'After closing other browser tabs and windows, try quitting and reopening the Trezor Suite app.',
```
Comment (tomasklim) [inline packages/suite/src/support/messages.ts:6362]: The easier solution is to reload the app the same way the browser can be reloaded. However, it is discutable if we want to inform a user that he can reload the app using ctrl/cmd + R

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/support/messages.ts (hunk 4 above)
Comment (marekrjpolak) [reply packages/suite/src/support/messages.ts:6362]: After discussion, we agreed to preserve the original variant.

**Review by tomasklim — APPROVED** [review]
LGTM

**Pull request merged**